src/main/jbake/content/overview008.adoc

```diff
@@ -118,106 +117,106 @@ The following features support the GUI components:
 * Data conversion between model objects and components
 * Managed model object creation
 * Page navigation configuration
-* Expression Language (EL)
+* Expression Language
 
 All this functionality is available using standard Java APIs and
 XML-based configuration files.
 
-In the Jakarta EE 8 platform, new features of JavaServer Faces technology
+In the Jakarta EE 8 platform, new features of Jakarta Server Faces technology
 include the following:
 
 * Direct support for WebSockets via the new `<f:websocket>` tag
 * Class-level bean validation via the new `<f:validateWholeBean>` tag
 * A CDI-compatible `@ManagedProperty` annotation
 * Enhanced component search expression framework
 
-The Jakarta EE 8 platform requires JavaServer Faces 2.3 and Expression
+The Jakarta EE 8 platform requires Jakarta Server Faces 2.3 and Jakarta Expression
 Language 3.0.
 
-For an excellent summary of what's new in JSF 2.3,
+For an excellent summary of what's new in Jakarta Server Faces 2.3,
 see `https://javaserverfaces.github.io/users.html`.
 
 [[BNACN]][[javaserver-pages-technology]]
 
-JavaServer Pages Technology
-~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Jakarta Server Pages Technology
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-JavaServer Pages (JSP) technology lets you put snippets of servlet code
-directly into a text-based document. A JSP page is a text-based document
+Jakarta Server Pages technology lets you put snippets of servlet code
+directly into a text-based document. A Jakarta page is a text-based document
 that contains two types of text:
 
 * Static data, which can be expressed in any text-based format, such as
 HTML or XML
 * JSP elements, which determine how the page constructs dynamic content
 
-For information about JSP technology, see _The Jakarta EE 5 Tutorial_ at
+For information about JSP technology, see _The Java EE 5 Tutorial_ at
 `http://docs.oracle.com/javaee/5/tutorial/doc/`.
```

Comment:
It's weird that Jakarta Server Pages is referring to an old Java EE tutorial, and it's
unfortunate that all the JSP content was removed from the current version of the
tutorial.  I think we at least need a sentence something like this at the beginning
of this paragraph... "The Jakarta Server Pages technology is derived from and compatible
with the JavaServer Pages (JSP) technology."

Reply:
Great suggestion, thanks